src/transitions/Collapse.d.ts

```diff
@@ -8,7 +8,6 @@ export interface CollapseProps
   children?: React.ReactNode;
   collapsedHeight?: string;
   component?: React.ReactType<CollapseProps>;
-  containerProps?: Object;
```

Comment:
Is this a bug that? The prop existed in JS but wasn't being used at all, so I removed it.